packages/zapp/console/src/components/Executions/Tables/NodeExecutionsTable.tsx

```diff
@@ -82,14 +117,6 @@ export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
       <NodeExecutionsTableContext.Provider value={tableContext}>
         <div className={tableStyles.scrollContainer}>{content}</div>
```

Comment:
```suggestion
        <div className={tableStyles.scrollContainer}>
          {executionsWithKeys.length > 0 ? (
            executionsWithKeys.map(({ nodeExecution, cacheKey }, index) => {
              return (
                <NodeExecutionRow
                  {...rowProps}
                  abortMetadata={abortMetadata}
                  index={index}
                  key={cacheKey}
                  execution={nodeExecution}
                />
              );
            })
          ) : (
            <NoExecutionsContent size="large" />
          )}
        </div>
```
and delete `content` var

Reply:
fixed